### PR TITLE
adds visual basic support

### PR DESCRIPTION
**Pull request by fg-j**
Signed-off-by: Forest Eckhardt <feckhardt@vmware.com>

## Summary

Adds support + test for Visual Basic apps

## Use Cases

Part of addressing paketo-buildpacks/dotnet-core#5

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have added an integration test, if necessary.

### CODE REVIEW

**Comment by fg-j** [comment]
Blocked on merge and release of https://github.com/paketo-buildpacks/dotnet-core-sdk/pull/129, which will break the `DOTNET_ROOT` behavior of this buildpack.

File: detect.go [hunk 1]
```diff
@@ -33,17 +34,15 @@ func Detect(parser ProjectParser, buildpackYMLParser BuildpackYMLParser) packit.
 			return packit.DetectResult{}, fmt.Errorf("failed to parse buildpack.yml: %w", err)
 		}
 
-		matches, err := filepath.Glob(filepath.Join(context.WorkingDir, projectPath, "*.?sproj"))
+		projectFilePath, err := parser.FindProjectFile(filepath.Join(context.WorkingDir, projectPath))
 		if err != nil {
-			return packit.DetectResult{}, err
+			panic(err)
```
Comment (ForestEckhardt) [inline detect.go:39]: @fg-j I don't think you want this here

Comment (fg-j) [reply detect.go:39]: but i love panicking...

**Comment by ForestEckhardt** [comment]
@fg-j I can't find the corresponding test for that panic. Am I just missing it?

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**